core/src/main/java/org/apache/calcite/rel/core/Sort.java

```diff
@@ -143,6 +143,14 @@ public abstract Sort copy(RelTraitSet traitSet, RelNode newInput,
     return fieldExps;
   }
 
+  public RexNode getFetch() {
+    return fetch;
+  }
+
+  public RexNode getOffset() {
+    return offset;
+  }
```

Comment:
Both `fetch` and `offset` are public. So it seems not necessary.

Reply:
@chunweilei I think it is still not a good idea to access the members directly. Also in future we can thinking of changing the access level in which case changes will be minimum and confined to the class itself.